views/pages/partials/artwork.ejs

```diff
@@ -1,7 +1,7 @@
 <div class = "artworkDiv">
     <% if (artwork.artworkImage) %>
-    <p class="artworkTitle"><%= artwork.artworkTitle%> by <%=artwork.artistName%></p>
```

Comment:
Title should be above image, not after it.

Reply:
moved